workers/tests/integrationtests/api_response_schema.py

```diff
@@ -49,19 +41,54 @@ def get_response_schema() -> Schema:
     )
 
 
-def get_feature_schema(iterator: Optional[str] = None):
+def get_result_schema() -> Schema:
+    """Response schema for all endpoints except of 'indicator' and 'report' endpoints"""
+    return Schema({"result": list}, ignore_extra_keys=True)
+
+
+def get_featurecollection_schema() -> Schema:
+    """Response schema for responses of type FeatureCollection"""
+    return Schema(
+        {"type": "FeatureCollection", "features": list}, ignore_extra_keys=True
+    )
+
+
+def get_indicator_feature_schema() -> Schema:
+    properties = get_indicator_properties_template()
     return Schema(
         {
             "type": "Feature",
             "geometry": dict,
             Opt("id"): Or(str, int),
-            "properties": generate_properties_template(iterator),
+            "properties": properties,
         },
         ignore_extra_keys=True,
     )
 
 
-def get_featurecollection_schema():
-    return Schema(
-        {"type": "FeatureCollection", "features": list}, ignore_extra_keys=True
+def get_report_feature_schema(number_of_indicators: int) -> Schema:
+    properties_template = get_indicator_properties_template()
+    properties = {}
+    for i in range(number_of_indicators):
+        for k, v in properties_template.items():
+            if k == Opt("result.svg"):
+                properties.update({Opt("indicators." + str(i) + "." + "result.svg"): v})
+            else:
+                properties.update({"indicators." + str(i) + "." + k: v})
```

Comment:
I'm not a fan of these lines. Do you know a better way to handle this? Maybe something like this:
```suggestion
            prefix = "indicators." + str(i) + "."
            if isinstance(k, Schema):
                k._prepend_schema_name(prefix)
            else:
                k = prefix + k
            properties.update({"indicators." + str(i) + "." + k: v})
```
But this is just a *suggestion*, not something that needs to be changed.